docs/v0.0.1/layout-specs.md

```diff
@@ -41,90 +82,182 @@ Each element in the hierarchical structure contains the following key-value pair
 
 ![image](img/layout-element-types.png)
 
-### Adding items to an element
-Data types can be added to a given element by adding them as values to `include` key, as illustrated in the example below. In this case, both `DSCore.Color` and `DSCore.ColorRange2D` classes will be added under `Display` library item.
+### Adding classes to an element
+Item classes can be added to a given element by adding them as values to `include` key, as illustrated in the example below. In this case, both `DSCore.Color` and `DSCore.ColorRange2D` classes will be added under `Display` library item. `iconURL` represent the absolute URL of the icons for the classes.
 
 ```json
 {
-  "elements": [
+  "sections": [
     {
-      "text": "Display",
-      "iconUrl": "Category.Display.svg",
-      "elementType": "category",
-      "include": [
-        { "path": "DSCore.Color" },
-        { "path": "DSCore.ColorRange2D" }
-      ],
-      "childElements": []
+      "text": "default",
+      "iconUrl": "",
+      "elementType": "section",
+      "showHeader": false,
+      "include": [],
+      "childElements": [
+        {
+          "text": "Display",
+          "iconUrl": "/src/resources/icons/Category.Display.svg",
+          "elementType": "category",
+          "include": [
+            {
+              "path": "DSCore.Color",
+              "iconUrl": "/src/resources/icons/DSCore.Color.png"
+            },
+            {
+              "path": "DSCore.ColorRange",
+              "iconUrl": "/src/resources/icons/DSCore.ColorRange.png"
+            }
+          ],
+          "childElements": []
+        }]
     }]
 }
 ```
+Note that this will create two elements with *none* as `elementType`, named "Color" and "ColorRange", each containing nodes with names that start with "DSCore.Color" and "DSCore.ColorRange" respectively:
```

Comment:
Can we use italic fonts for `Color` and `ColorRange`? Then use fixed-width fonts for `DSCore.Color` and `DSCore. ColorRange `?